pom.xml

```diff
@@ -91,17 +91,18 @@
     <governator.version>1.17.4</governator.version>
     <jackson.version>2.9.2</jackson.version>
```

Comment:
/docs/pom.xml, Can you update sphinx-maven-plugin to 1.7.0. 1.6.1.Final doesn't exist at https://repo.maven.apache.org/maven2/kr/motd/maven/sphinx-maven-plugin/

Reply:
https://github.com/cerner/beadledom/commit/66d28e32652a3d9b720293986ee67e0fead66a99?diff=unified#diff-48fabae9c2e321b2ab4d99057d5dc32eL46